src/libraries/Common/src/System/Security/Cryptography/RSAAndroid.cs

```diff
@@ -825,7 +825,7 @@ public override bool VerifyHash(ReadOnlySpan<byte> hash, ReadOnlySpan<byte> sign
                 {
                     throw new ArgumentNullException(nameof(padding));
                 }
-                if (padding == RSASignaturePadding.Pkcs1 && padding == RSASignaturePadding.Pss)
+                if (padding != RSASignaturePadding.Pkcs1 && padding != RSASignaturePadding.Pss)
```

Comment:
I assume we have tests for this and they're just not yet running on Android? (Or maybe we're scouting them and that's how we found this?)